common/models/link_tag.go

```diff
@@ -0,0 +1,28 @@
+package models
+
+type LinkTags []string
+
+func (l LinkTags) Contains(value string) bool {
+	for _, v := range l {
+		if v == value {
+			return true
+		}
+	}
+
+	return false
+}
+
+func (l LinkTags) Distinct() []string {
+	m := map[string]int{}
+
+	for idx, k := range l {
+		m[k] = idx
+	}
+
+	d := []string{}
```

Comment:
[super nit] for future reference, if we needed to be super high perf we'd use `d := make([]string, len(m))` and use `d[i] = k` instead of `append`

Reply:
oops - i commented on @nehayward review about the same thing yesterday and then ended up doing the same thing myself. will fix ✅